internal/transport/transport.go

```diff
@@ -355,6 +365,18 @@ func webWorker(ctx context.Context, workerID int, jobs <-chan *webJob) {
 			job.logger.Fatal(err)
 		}
 
+		if !json.Valid(bytes) {
+			if job.flattenedRequest.clobColumn == "" {
+				job.repoJobs <- nil
+				job.logger.Warnf("response body for %s was HTML, discarding data since no 'clobColumn' was defined in the configuration file", job.fetchConfig.URL)
```

Comment:
So this checks to see if the data is valid JSON and then inputs the data into the "clob" column. It's a bit more generic then just "HTML" but I think that's totally fine. Lets make this warning less specific so we catch any non-json type in the clob!
```suggestion
				msg := fmt.Sprintf("response body for %s was invalid JSON, " +
				    "discarding data since no 'clobColumn' was defined in the configuration file",
				     job.fetchConfig.URL)
			        logInfo := tools.LogFormatter{Msg: msg}
		                cfg.Logger.Warnf(logInfo.String())
```